src/apps/sketch/components/Board/board.jsx

```diff
@@ -26,14 +28,34 @@ const Board = props => {
       setColor(c);
   }
 
+  const increaseFont = () => {
+    setFont(prevFont => {
+      return prevFont < MAX_FONT? prevFont + 1: prevFont;
+    })
+  }
+  
+  const reduceFont = () => {
+    setFont(prevFont => {
+      return prevFont > MIN_FONT? prevFont - 1 : prevFont;
+    })
+  }
+
+  const paletteHandler = {
+    font: handleFont,
+    color: handleColor,
+    getColor: () => { return color },
+    increaseFont: increaseFont,
+    reduceFont: reduceFont,
+  }
+
   return (
     <Row>
       <Col xs={24} xl={20}>
         {
           brush 
             ? <>
-                <SketchBoard brush = {brush} font = {font} color = {color}/>
-                <Buttons handleFont={handleFont} handleColor={handleColor}/>
+                <SketchBoard brush = {brush} font = {font} color = {color} paletteHandler = {paletteHandler} />
+                <Palette handleFont={handleFont} handleColor={handleColor}/>
```

Comment:
better to change the button file name also, I see that it is defined in buttons.jsx file

Reply:
correcting